### ISSUE TITLE
Add --short flag

### ISSUE THREAD

**Issue by zakird** [zmap/zdns#130]
Right now, ZDNS output is very verbose. This is useful when doing longterm scheduled tasks, but is really quite annoying when just trying to play with data on the command line. For example, this is a single reverse DNS lookup:

```json
{
  "name": "104.73.21.236",
  "class": "IN",
  "status": "NOERROR",
  "timestamp": "2017-09-14T15:19:00-04:00",
  "data": {
    "answers": [
      {
        "ttl": 43200,
        "type": "PTR",
        "class": "IN",
        "name": "236.21.73.104.in-addr.arpa",
        "answer": "a104-73-21-236.deploy.static.akamaitechnologies.com."
      }
    ],
    "additionals": [
      
    ],
    "authorities": [
      
    ],
    "protocol": "udp",
    "flags": {
      "response": true,
      "opcode": 0,
      "authoritative": false,
      "truncated": false,
      "recursion_desired": true,
      "recursion_available": true,
      "authenticated": false,
      "checking_disabled": false,
      "error_code": 0
    }
  }
}
```

We should also support a "terse" mode that looks closer to:

```json
{
  "name": "104.73.21.236",
  "status": "NOERROR",
  "timestamp": "2017-09-14T15:19:00-04:00",
  "data": {
    "answers": [
      {
        "ttl": 43200,
        "type": "PTR",
        "name": "236.21.73.104.in-addr.arpa",
        "answer": "a104-73-21-236.deploy.static.akamaitechnologies.com."
      }
    ]
  }
}
```

Arguably we could even have a --very-terse:

```json
{
  "name": "104.73.21.236",
  "status": "NOERROR",
  "timestamp": "2017-09-14T15:19:00-04:00",
  "answers": ["a104-73-21-236.deploy.static.akamaitechnologies.com." ]
}
```

I suspect what we really want is `--terse` and `--verbose` flags. "Terse" from above becomes the default, "Verbose" becomes what's currently in place, and "Terse" becomes the last example.

**Comment by zakird** [comment]
Suggestion from @paul-pearce : call it `--short` not `--terse`

**Comment by zakird** [comment]
note: we should consider how this interacts with `--trace`. Arguably, that' just another verbosity level.

**Comment by paul-pearce** [comment]
Chiming in.

* As discussed, I think this should be called short. Everyone already knows what that means from dig
* I think the default should be what's described as "terse" above
* I think the _current_ default should become `verbose`
* We'll need to move the current verbosity flags (which really just outputs debugging info) to `debug` or similar
* I'd ideally like `short` to mimic the dig output for short, but that doesn't work for N items performed in "random" order. So I'd argue that the `short` output should be "very-terse" above.

**Comment by dadrian** [comment]
Preface: I'm not actually that familiar with ZDNS.

 - I suspect the current verbosity flags is really just a `--log-level`, which is separate from the actual verbosity of the ZDNS output, and should only affect the `--log-file`. If this isn't already the case, we should move to a model where it is, and have debug info, etc all channeled to the log file.
- The variants that zakir described I think fall under a `--verbosity` flag. However, I also recognize the value of having `--short` et al options. I would suggest doing the following:
    1. Figure out how many levels you have. Based on Zakir's comment, it looks like there are 3.
    2. Implement them as `--verbosity={0, 1, 2}` 
    3. Give all or some of them a human-readable name, e.g. level `0` is `short` and level 2 is `detailed`. Also accept `--verbosity=short` or `--verbosity=long`. Map them correctly internally.
    4. For the named verbosity levels we care about, namely `short`, also allow the shorthand `--short`. Pick some arbitrary precedence level between passing `--short`, `--long`, and `--verbosity=1` all at the same time (or error if someone does this). I'd probably have `--verbosity` override any of the short hands.

This will probably be real annoying to implement the command-line handling for, but I think it's a nice user interface, and I think I've seen other projects do this (maybe either rsync or sshd?).

Or maybe I'm just crazy 😝

**Comment by zakird** [comment]
This is resolved by #184.
